azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/service/IWebAppDeploymentSlot.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.toolkits.appservice.service;
+
+import com.microsoft.azure.toolkits.appservice.entity.WebAppDeploymentSlotEntity;
+import com.microsoft.azure.toolkits.appservice.model.DeployType;
+
+import java.io.File;
+
+public interface IWebAppDeploymentSlot {
+    IWebAppDeploymentSlotCreator create();
+
+    void start();
+
+    void stop();
+
+    void restart();
+
+    void delete();
+
+    void deploy(File file);
+
+    void deploy(DeployType deployType, File file);
+
+    boolean exists();
```

Comment:
what's the meaning of "exists()" ? Is it "equals to "get() = null" ?

Reply:
get() will return the entity directly, but exists() will check whether the resource exists in Azure